src/components/CabinList/index.js

```diff
@@ -1,21 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cabinData from '../../data/cabin';
 
 function CabinList() {
   const click = (data) => {
+    setCabinName(data);
     console.log(data);
   };
+  const [cabinName, setCabinName] = useState('Select Cabin');
   const cabinNameList1 = cabinData.map((data) => data.cabin);
   const cabinNames = [...new Set(cabinNameList1)];
   const cabin = cabinNames.map((data, index) => (
     <li className="cabinNameList" key={index}>
-      <button className="cabinListBtn" key={index} onClick={() => click(data)}>
+      <button
+        className={data === cabinName ? 'cabinListBtnSelected' : 'cabinListBtn'}
+        key={index}
```

Comment:
oh one last thing - RE the 'key'.  I think if you look in the console you'll see that you have an error about duplicate keys.  Keys are a bit tricky to get your head around, but, in summary, you only need to have a key on your LI here, you can remove it from your button.

If you do want to keep it on your button, you can do, but you need to make it different to the key you use on the LI (as currently they're duplicated).

So either:
1. remove the button key
2. or, change key on the button to something unique like ```key={`button${index}`}```